cmd/ocm/fail/job/job.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package job
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+)
+
+// Cmd Constant:
+var Cmd = &cobra.Command{
+	Use:   "job QUEUE_NAME JOB_ID RECEIPT_ID FAILURE_REASON",
+	Short: "Mark the Job as a failure",
+	Long:  "Mark the Job as a failure with specific reason on the specified Job Queue.",
+	Args:  cobra.ExactArgs(4),
+	RunE:  run,
+}
+
+func run(_ *cobra.Command, argv []string) error {
+	var err error
+
+	// Create the client for the OCM API:
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	defer connection.Close()
+
+	// Get the client for the Job Queue management api
+	client := connection.JobQueue().V1()
+
+	// Send a request to Success a Job:
+	_, err = client.Queues().Queue(argv[0]).Jobs().Job(argv[1]).Failure().ReceiptId(argv[2]).FailureReason(argv[3]).Send()
```

Comment:
missing validation for argv items existance before calling them

Reply:
There is no way to validate argv items, but to count them, so there is `cobra.ExactArgs(4)`